app/src/main/java/com/devshish/internship/presentation/ui/player/PlayerFragment.kt

```diff
@@ -58,6 +60,19 @@ class PlayerFragment : Fragment(R.layout.fragment_player) {
             currentPosition.observe(viewLifecycleOwner) { position ->
                 binding.sPlayer.progress = position
             }
+
+            searchSong.observe(viewLifecycleOwner) { song ->
+                binding.ivLyrics.isVisible = song != null
+
+                binding.ivLyrics.setOnClickListener { onLyricsClick(song) }
```

Comment:
3. and then you can make click on this button out of observe, and remove parameters